src/main/flight/pid.c

```diff
@@ -438,6 +438,90 @@ static float accelerationLimit(int axis, float currentPidSetpoint)
     return currentPidSetpoint;
 }
 
+static FAST_RAM timeUs_t crashDetectedAtUs;
```

Comment:
Please be mindful about spontaneous uses  of `FAST_RAM` storage specifier, most of the time the performance gains are not-trivial to measure or even result in performance decrease.
If it's a refactor, better leave this variable in ordinary SRAM.
Meanwhile I'm working on optimizing F7 and partially F4.

Reply:
Ok fair enough to keep it same for refactoring work